### ISSUE TITLE
Readme should be updated with working example

### ISSUE THREAD

**Issue by michalzubkowicz** [shakilsiraj/json-object-mapper#32]
Example from Readme doesn't work properly with TypeScript. 

It should be updated or have info about that all classess used by object mapper must be created with constructor or marked all fields as undefined

**Comment by artonio** [comment]
We use this in production so this library works in most cases, however it does not seem to be maintained. None of my pull requests are being reviewed :(

**Comment by shakilsiraj** [comment]
Sorry guys, I have been busy with other things .. didn't look at this for a while..

@artonio .. see release 1.6.0
@michalzubkowicz - that's a requirement for typescript and there are plenty of examples in the spec files.

**Comment by t-gawel-samsung** [comment]
Deserialization does not work. I got empty object. Default settings, angular 6 project.

**Comment by michalzubkowicz** [comment]
@t-gawel-samsung , did you set all properties of object as undefined explictly?

**Comment by t-gawel-samsung** [comment]
@michalzubkowicz yes, I finally found it is a fix. But this information is missing in the docs, or I just overlooked it?

**Comment by michalzubkowicz** [comment]
@t-gawel-samsung - yes, it's missing in docs. It's in spec files, but nice would be to have it in example as I mentioned before.
